packages/opentelemetry-core/src/baggage/propagation/W3CBaggagePropagator.ts

```diff
@@ -58,13 +58,14 @@ export class W3CBaggagePropagator implements TextMapPropagator {
   }
 
   extract(context: Context, carrier: unknown, getter: TextMapGetter): Context {
-    const headerValue: string = getter.get(carrier, BAGGAGE_HEADER) as string;
-    if (!headerValue) return context;
+    const headerValue = getter.get(carrier, BAGGAGE_HEADER);
+    const baggageString = Array.isArray(headerValue) ? headerValue.join(',') : headerValue;
```

Comment:
Can we replace this hard-coded `','` with a meaningful constant?

```suggestion
    const baggageString = Array.isArray(headerValue) ? headerValue.join(BAGGAGE_ITEMS_SEPARATOR) : headerValue;
```